lib/src/options.dart

```diff
@@ -110,8 +110,11 @@ class RoomOptions {
   final bool stopLocalTrackOnUnpublish;
 
   /// Options for end-to-end encryption.
+  @Deprecated('Use encryption instead')
```

Comment:
we'll want some more info here in the deprecation notice, pointing to a guide (that I'll write today) that outlines the differences